pandas/core/ops/docstrings.py

```diff
@@ -33,9 +33,14 @@ def _make_flex_doc(op_name, typ):
             desc=op_desc["desc"],
             op_name=op_name,
             equiv=equiv,
-            reverse=op_desc["reverse"],
             series_returns=op_desc["series_returns"],
         )
+        if op_desc["reverse"]:
+            doc_see_also = _see_also_reverse_SERIES.format(
+                reverse=op_desc["reverse"],
+                see_also_desc=op_desc["see_also_desc"],
+            )
+            doc_no_examples = doc_no_examples + doc_see_also
```

Comment:
Do you mind opening a separate pull request for this? I think the rest can be merged already, and this is a bit trickier.